src/Tools/babylon.tools.ts

```diff
@@ -698,6 +698,12 @@
             return false;
         }
 
+        public static getClassName(obj): string {

```

Comment:
Well, I don't care about the class' Name if it changes due to minification, I mean I won't do a getClassName() == MyClass. I just want to built a tree of the class hierarchy and add metadata on them. So typically I call this method to store the name of the class and then later to find it from an instance.
So maybe it could work in my case, right?